Automattic/jetpack
# Dialogue: Connect podcast player with dialogue front-end

This PR adds the code to play the podcast player when the user clicks on the timestamp label button.

Fixes #

#### Changes proposed in this Pull Request:

* Dialogue: Connect podcast player with dialogue front-end

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



no

#### Testing instructions:






* Create/edit a post
* Add a podcast player with a valid RSS feed. For instance, https://distributed.blog/2020/10/15/episode-25-davit-baghdasaryan-on-the-science-of-sound-in-a-distributed-work-world/
* Add a transcription block
* Show timestamp in the Dialogue blocks.
* Set a custom time.
* Save the post
* Preview/see the post in the front-end
* Click on the ~timestamp label~ play/pause button.
* Confirm the player plays in the timestamp position.

#### Proposed changelog entry for your changes:


*
